cryptofeed/exchange/okcoin.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, **kwargs):
             if chan != LIQUIDATIONS:
                 continue
             for symbol in self.subscription[chan]:
-                instrument_type = self.instrument_type(symbol)
+                instrument_type = self.instrument_type(self.exchange_symbol_to_std_symbol(symbol))
```

Comment:
This was a bug in the old system, it just didn't cause issues since the exchange and standard symbols were the same for non-index symbols.